arch/arm/core/fault.c

```diff
@@ -268,8 +290,13 @@ static u32_t _MpuFault(const NANO_ESF *esf, int fromHardFault)
  *
  * @return N/A
  */
-static void _BusFault(const NANO_ESF *esf, int fromHardFault)
+static int _BusFault(NANO_ESF *esf, int fromHardFault)
 {
+	/* Assess whether system shall ignore/recover from this MPU fault. */
```

Comment:
I suggest to put this inside #ifdef NXP_MPU,  as this is only relevant for NXP SOCs that generate BusFaults for memory access violations.

Reply:
This isn't true.
For example, in the provided test case, user mode passes a nonsense string address of 0xFFFFFFF0. This generates a bus fault.

Reply:
Ah, I see this possibility now, thanks for pointing it out, @andrewboie